components/aiprompt/overview.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Overview
+page_title: AIPrompt Overview
+description: Discover the AIPrompt component for Blazor. Learn how to add the component to your app and explore its features like handling prompts and their outputs.
+slug: aiprompt-overview
+tags: telerik,blazor,aiprompt,ai,prompt
+published: True
+position: 0
+---
+
+# Blazor AIPrompt Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/aiprompt" target="_blank">Blazor AIPrompt component</a> helps you write prompts, look and interact with the output from the AI, and execute a set of predefined commands. The AIPrompt comes with three predefined views - Prompt, Output, and Command, as well as the option to define custom views. Each view can be navigated through the AIPrompt's toolbar.
```

Comment:
It may be good to define the `prompt` word somewhere in the beginning, as it's an important term and we want to be sure the reader knows what it is (e.g. I didn't and had to Google it).

The `look` word doesn't fit. Consider `receive` or just remove it.